home/import_content_pages.py

```diff
@@ -491,34 +494,60 @@ def validate_page_using_form(self, page: Page) -> None:
                     del errs["slug"]
             # TODO: better error stuff
             if errs:
-                error_message = self.errors_to_strings(errs)
-                raise ImportException(
-                    f"Validation error: {error_message}", self.row_num
-                )
+                errors = []
+                error_message = self.errors_to_list(errs)
+                for err in error_message:
+                    errors.extend(ValidationError(f"Validation error: {err}"))
```

Comment:
Do we want this to be a `ValidationError`? We're expecting strings in `ImportException`. Also, since we're not passing in an iterable, we need `.append()` rather than `.extend()`.